features/support/env.rb

```diff
@@ -17,4 +17,16 @@
 require 'cucumber/rails'
 Capybara.default_selector = :css
 ActionController::Base.allow_rescue = false
+
+require 'database_cleaner/active_record'
 DatabaseCleaner.strategy = :truncation
+
+Around do |scenario, block|
+  DatabaseCleaner.cleaning(&block)
+end
```

Comment:
Not sure why this wasn't included before.
This is normal configuration for DatabaseCleaner https://github.com/DatabaseCleaner/database_cleaner?tab=readme-ov-file#cucumber-example

Without these, database isn't cleaned between cucumber scenarios.

Reply:
it might have been removed if its not necessary, because when cleaning is not required, its infinitely faster NOT to do something than any speed gain.

Reply:
We actually rely on data to be removed in some tests, like here: https://github.com/ianwhite/pickle/blob/master/features/pickle/create_from_machinist.feature#L21-L32

In the beginning of this scenario, we already will have 3 spoon records created, so on line 32 it will be an error (because we will have 5 "non-round" spoons at the moment)

Thanks for review!